### PR TITLE
Documentation for afterCreate pool config

### PR DESCRIPTION
**Pull request by elhigu**
Fixes https://github.com/tgriesser/knex/issues/1364 and https://github.com/tgriesser/knex/issues/1621

### CODE REVIEW

**Comment by elhigu** [comment]
Merging this already, since this is just missing documentation for already implemented feature

File: sections/installation.js [hunk 1]
```diff
@@ -187,6 +187,44 @@ export default [
...
+        pool: {
+          afterCreate: function (conn, done) {
+            // in this example we use pg driver's connection API
+            conn.query('SET timezone="UTC";', function (err) {
+              if (err) {
+                // first query failed, return error and don't try to make next query
+                done(err, conn);
+              } else {
+                // do the second query...
+                conn.query('SELECT set_limit(0.01);', function (err) {
```
Comment (raghavgarg1257) [inline sections/installation.js]: can you please explain, why we should test the connection with two queries?

Comment (elhigu) [inline sections/installation.js]: There is no need to make even a single query in `afterConnect`. Connection is already made when the callback is executed.

Only reason why in this example there are 2 queries is to show how multiple queries is done with that ugly and error prone callback syntax. As you can see those queries are just some setup queries how the connection which was just acquired should behave.

Comment (raghavgarg1257) [inline sections/installation.js]: Oh, okay now I get it. Thanks for the reply.

**Pull request merged**